### PR TITLE
Document using travix.Logger conditionally

### LINKED ISSUE #58: Using travix.Logger only for travis builds
Documentation recommends to use travix.Logger over trace() and Sys.exit()

Does travix set a compiler flag when invoking the Haxe compiler so I could use a conditional in my classes? For example:

```haxe
#if travix
    travix.Logger.trace(...);
#else
    trace(...):
#end
```

### PR DESCRIPTION
**Pull request by sebthom**
Resolves #58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,14 @@ Use them to instead of `trace()`, `Sys.exit()`, etc, for maximum compatibility a
 - `travix.Logger.println(string)`: Print a string with newline
 - `travix.Logger.exit(exitCode)`: Exit the process
 
+If you want to be able to run your tests e.g. locally without Travix dependency use conditional compilation like this:
+```haxe
+#if travix
```
Comment (back2dos) [inline README.md]: FWIW `#if travix import travix.Logger.println in trace; #end` keeps the conditional compilation more limited. 

That said, I do wonder why you wouldn't want to use travix locally. Having two different ways to run the tests adds complexity.

Comment (kevinresol) [inline README.md]: > That said, I do wonder why you wouldn't want to use travix locally. Having two different ways to run the tests adds complexity.

+1

Comment (sebthom) [inline README.md]: Probably only relevant for haxelibs. I added Travis/travix support for my [https://github.com/vegardit/haxe-doctest] library but don't want to add a hard but optional dependency on travix.

**Comment by kevinresol** [comment]
btw this is how I do it: https://github.com/haxetink/tink_testrunner/blob/d762f78/src/tink/testrunner/Reporter.hx#L168-L177

**Comment by sebthom** [comment]
So then why not document it as a best-practice/tip?

This was one of the first questions I asked my self when I scanned the readme. How to prevent introducing a hard dependency to travix in a haxelib.

**Comment by kevinresol** [comment]
No, in no way I mean it shouldn't go to the readme.

But, at least it should be correct: `Logger` does not define the method `trace`

**Comment by sebthom** [comment]
Ah ok. My PR does not do so. But I can adjust the example or description as required. It is just a suggestion.

**Pull request closed without merging**